client/packages/system/src/Patient/CreatePatientModal/DefaultCreatePatientUISchema.json

```diff
@@ -3,21 +3,21 @@
   "elements": [
     {
       "type": "Group",
-      "label": "Patient Information",
+      "label": "heading.patient-details",
```

Comment:
thought could use the T# used by backend schema, but obviously these schema are not processed through backend! So just supplying translation keys, and using from renderers